src/tests/ftest/deployment/ior_per_rank.py

```diff
@@ -33,52 +33,46 @@ def execute_ior_per_rank(self, rank):
         # create the pool on specified rank.
         self.add_pool(connect=False, target_list=[rank])
```

Comment:
I noticed that creating a container outside the IOR doesn't cause pool destroy failures. I think I tried something like this:

self.add_pool(connect=False, target_list=[rank])
self.add_container(self.pool) 

.... On IOR commands create_cont is always set False. 
            try:
                self.ior_cmd.transfer_size.update(transfer_size)
                self.ior_cmd.flags.update(self.write_flags)
                dfs_out = self.run_ior_with_pool(create_cont=False,fail_on_warning=self.log.info)
                dfs_perf_write = IorCommand.get_ior_metrics(dfs_out)
                self.ior_cmd.flags.update(self.read_flags)
                dfs_out = self.run_ior_with_pool(create_cont=False, fail_on_warning=self.log.info)

In this way, I didn't notice pool_destroy having problem on large scale. It looks like creating containers for each rank and destroying them and pool has some issues on large scale testing.

Reply:
Ah! Seems another issue with pydaos handling. If we let `run_ior_with_pool` create the container, it calls `pool.connect()`. I'll have the test create it here, which I think is a better workflow anyway. Thanks for finding that!